test/test_trixi2vtk.jl

```diff
@@ -53,11 +53,7 @@ end
 # Note: The purpose of using a specific commit hash (instead of `main`) is to be able to tie a given
 #       version of Trixi2Vtk to a specific version of the test file repository. This way, also tests
 #       for older Trixi2Vtk releases should continue to work.
-if VERSION < v"1.8"
-  const TEST_REFERENCE_COMMIT = "c0a966b06489f9b2ee3aefeb0a5c0dae733df36f"
-else
-  const TEST_REFERENCE_COMMIT = "e51f3613ac1adfcfd2bf1d74a2756034dab0579c"
-end
+const TEST_REFERENCE_COMMIT = "e51f3613ac1adfcfd2bf1d74a2756034dab0579c"
```

Comment:
Does it still make sense to keep testing with a specific commit hash instead of `main`?

Reply:
Yes, I think the reasoning in the comment still holds, don't you?